packages/widget-reorder/src/control.tsx

```diff
@@ -90,10 +95,10 @@ export const createControl: CreateControl = (options = {}) => {
                 {...provided.droppableProps}
                 ref={provided.innerRef}
               >
-                {data.map((item, i) => (
+                {renderData.map((item, i) => (
                   <Draggable
-                    key={item[fieldId]}
-                    draggableId={item[fieldId]}
+                    key={`draggable${i}`}
+                    draggableId={`draggable${i}`}
```

Comment:
We'll want to avoid using array index as key here, since they'll be changed every re-order. The `index` prop should still be `i`, but the `key` & `draggableId` should be unique to the item! We need `item[fieldId]` here

Reply:
Ahh, didn't think about that! Good call!